src/SWP/Bundle/ContentBundle/Loader/ArticleLoader.php

```diff
@@ -78,7 +80,7 @@ public function load($type, $parameters, $responseType = LoaderInterface::SINGLE
                     ->findOneBy(['slug' => $parameters['slug']]);
             }
 
-            if (!is_null($article)) {
+            if (!is_null($article) && $publishWorkflowChecker->isGranted(PublishWorkflowChecker::VIEW_ATTRIBUTE, $article)) {
```

Comment:
Better to put this logic in a getMeta($article) method instead of copy and pasting it